GeorgianMemenator/Network/BusinesssModels/DogBreed.swift

```diff
@@ -7,10 +7,14 @@
 //
 
 import Foundation
-
```

Comment:
We need new line between import and struct definition

Reply:
I have added, but you could not see as it seems, I dont know why

Reply:
<img width="555" alt="Screenshot 2023-02-26 at 15 46 27" src="https://user-images.githubusercontent.com/111251425/221408619-df1cc9be-615e-42f1-b3bb-548800fb616e.png">

Reply:
You attached the screenshot for `DogBreedDTO`, but I am talking about `DogBreed`